mm/2s2h/Rando/StaticData/Checks.cpp

```diff
@@ -144,6 +144,8 @@ std::map<RandoCheckId, RandoStaticCheck> Checks = {
     RC(RC_MAYORS_OFFICE_PIECE_OF_HEART,                                  RCTYPE_NPC,              SCENE_SONCHONOIE,               FLAG_WEEK_EVENT_REG,         WEEKEVENTREG_BOMBERS_NOTEBOOK_EVENT_RECEIVED_MAYOR_HP,               RI_HEART_PIECE),
     RC(RC_MILK_BAR_CIRCUS_LEADER_MASK,                                   RCTYPE_NPC,              SCENE_MILK_BAR,                 FLAG_NONE,                   0x00,                                                                RI_MASK_CIRCUS_LEADER),
     RC(RC_MILK_BAR_MADAME_AROMA,                                         RCTYPE_NPC,              SCENE_MILK_BAR,                 FLAG_WEEK_EVENT_REG,         WEEKEVENTREG_57_04,                                                  RI_BOTTLE_CHATEAU_ROMANI),
+    RC(RC_MILK_BAR_PURCHASE_CHATEAU,                                     RCTYPE_SHOP,             SCENE_MILK_BAR,                 FLAG_NONE,                   0x00,                                                                RI_CHATEAU_ROMANI_REFILL),
+    RC(RC_MILK_BAR_PURCHASE_MILK,                                        RCTYPE_SHOP,             SCENE_MILK_BAR,                 FLAG_NONE,                   0x00,                                                                RI_MILK_REFILL),
```

Comment:
Let's keep it this way for now, but we should gauge feedback on these being wrapped in with the Shop checks going forward.